ras_party/controllers/party_controller.py

```diff
@@ -81,15 +81,15 @@ def get_party_by_id(sample_unit_type, id, session):
         raise BadRequest(f"{sample_unit_type} is not a valid value for sampleUnitType. Must be one of ['B', 'BI']")
 
 
-def get_business_with_respondents_filtered_by_survey(sample_unit_type, id, survey_id):
+def get_business_with_respondents_filtered_by_survey(sample_unit_type, id, survey_id, enrolment_status=None):
```

Comment:
`id` here is this party id? business id? survey id? Its just named id all the way through from the view to where it needs to use it. Can we be explicit please

Reply:
business id it should be

Reply:
In this context this can be business party id or respondent party id but I will change it to party id

Reply:
oh really? I couldn't get it to work with a party id

Reply:
It should work with respondent party id with BI as sample unit type

Reply:
ah i wasn't trying BI as it wasn't in the DB. Ill give that a go